notebooks/cmb_inflation/Correlation.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p><a href="https://github.com/camilo1704" target="_blank">@camilo1704</a> you are comparing nominal values (delta of monetary base) vs a percentage (inflation over time). you need to do the comparison in percentage for the two.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lambdaclass/finance_playground/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
let's talk about this with @jrchatruc and @metdinov on monday

Reply:
@martinacantaro check the notebook too please.

Reply:
use pct_change https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.pct_change.html